docs/source/topics/proc_starting-monitoring-alerting-telemetry.adoc

```diff
@@ -20,22 +20,22 @@ For more information, see <<configuring-the-virtual-machine_{context}>>.
 
 ** On Linux or {mac}:
 +
-[subs="+quotes,attributes"]
+[subs="+quotes"]
```

Comment:
These ",attributes" removals seem unrelated, are they intentional?

Reply:
Yes, they're intentional, but only because I happened to notice them while building the docs to test how they build with the modules moved around.

The inclusion of "attributes" here causes warnings while building the docs about undefined attributes (the `{end}` portions of the source blocks are interpreted as attributes by AsciiDoctor).

I can move these changes into their own dedicated commit if you'd prefer.

Reply:
I don't feel that strongly about the separate commit (even if that's better ;), I wanted to make sure this was not an oversight.